src/utils/constants/tooltips_constants.ts

```diff
@@ -71,6 +71,39 @@ export const TOOLTIP_KEYS = {
   PORT: "Port",
   SWITCHING_TABLE: "Switching Table",
   MULTI_EDGE_CONNECTED_DEVICES: "Multi Edge Connected Devices",
+  ETHERTYPE: "EtherType",
+  // ARP Details
+  HTYPE: "HTYPE",
+  PTYPE: "PTYPE",
+  HLEN: "HLEN",
+  PLEN: "PLEN",
+  OP: "OP",
+  SHA: "SHA",
+  SPA: "SPA",
+  THA: "THA",
+  TPA: "TPA",
```

Comment:
Can we rename these?

```suggestion
  HTYPE: "Hardware Type",
  PTYPE: "Protocol Type",
  HLEN: "Hardware address Length",
  PLEN: "Protocol address Length",
  OP: "Operation code",
  SHA: "Sender Hardware Address",
  SPA: "Sender Protocol Address",
  THA: "Target Hardware Address",
  TPA: "Target Protocol Address",
```